data-sources/nvidia-dcgm/config.yml

```diff
@@ -0,0 +1,13 @@
+id: nvidia-dcgm
+displayName: NVIDIA DCGM
+description: |
+  Monitor and analyze your NVIDIA DCGM infrastructure with New Relic.
+install:
+  primary:
+    link:
+      url: https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/nvidia-dcgm-integration/
+icon: logo.png
+keywords:
```

Comment:
For this to show up in the Add data page, you'll need to add a keyword of `NR1_addData`